src/core_mapfiles/ecl.rs

```diff
@@ -380,38 +382,38 @@ static ECL_07: &'static CoreSignatures = &CoreSignatures {
         (Th07, 127, Some(("", None))),
         (Th07, 128, Some(("S", None))),
         (Th07, 129, Some(("SS", None))),
-        (Th07, 130, Some(("S", None))),
+        (Th07, 130, Some(("U", None))), // zero: b---
         (Th07, 131, Some(("ffSSSS", None))),
-        (Th07, 132, Some(("S", None))),
+        (Th07, 132, Some(("U", None))), // zero: b---
         (Th07, 133, Some(("", None))),
         (Th07, 134, Some(("", None))),
-        (Th07, 135, Some(("S", None))),
-        (Th07, 136, Some(("S", None))),
-        (Th07, 137, Some(("S", None))),
-        (Th07, 138, Some(("SSSS", None))),
-        (Th07, 139, Some(("SSSS", None))),
+        (Th07, 135, Some(("U", None))), // zero: b---
+        (Th07, 136, Some(("U", None))), // zero: b---
+        (Th07, 137, Some(("U", None))), // zero: b---
+        (Th07, 138, Some(("USSS", None))), // zero: b---SSS
+        (Th07, 139, Some(("SSSC", None))),
         (Th07, 140, Some(("ffff", None))),
-        (Th07, 141, Some(("S", None))),
+        (Th07, 141, Some(("S", None))), // Not implemented
         (Th07, 142, Some(("S", None))),
         (Th07, 143, Some(("f", None))),
         (Th07, 144, Some(("SE", None))),
         (Th07, 145, Some(("SS", None))),
         (Th07, 146, Some(("", None))),
         (Th07, 147, Some(("S", None))),
         (Th07, 148, Some(("SSE", None))),
-        (Th07, 149, Some(("SffS", None))),
+        (Th07, 149, Some(("Ufff", None))),
```

Comment:
woops? I wonder how this was accidentally `S` instead of `f`...

Reply:
Because it's a Z axis coordinate and always 0. Hard to tell when looking at decompiled files.